packages/atomic/src/components/common/generated-answer/generated-content/generated-markdown-content.spec.ts

```diff
@@ -0,0 +1,118 @@
+import DOMPurify from 'dompurify';
+import {html} from 'lit';
+import {afterEach, describe, expect, it, vi} from 'vitest';
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {
+  type GeneratedMarkdownContentProps,
+  renderGeneratedMarkdownContent,
+} from './generated-markdown-content';
+import * as markdownUtils from './markdown-utils';
```

Comment:
mocking of markdown-utils [1/4]

Import the function you need to mock or spy.

```suggestion
import {transformMarkdownToHtml} from './markdown-utils';
```